connector/connect/src/main/protobuf/spark/connect/relations.proto

```diff
@@ -192,5 +192,9 @@ message Sample {
   double lower_bound = 2;
   double upper_bound = 3;
   bool with_replacement = 4;
-  int64 seed = 5;
+  Seed seed = 5;
+}
+
+message Seed {
```

Comment:
I need to define `Seed` out of `Sample`, otherwise there is no `HasSeed` method in the generated files

Reply:
This is not true. The has* messages are generated for non simple types.

Reply:
you are right, maybe the jars were out of sync in that time, let me move `Seed` in `Sample`

Reply:
Yeah I always does a clean then build.